include/fastdds-statistics-backend/StatisticsBackend.hpp

```diff
@@ -224,8 +224,8 @@ class FASTDDS_STATISTICS_BACKEND_DllAPI StatisticsBackend
      * \sa StatisticsBackend
      *
```

Comment:
Explain in comment that these ids must belong to an entity related with this data type, and not any entity